examples/plugins/engine/examplePluginEngineWrite.cpp

```diff
@@ -84,8 +84,10 @@ int main(int argc, char *argv[])
         }
         else
         {
+            writer.BeginStep();
             writer.Put<float>(var, myFloats.data());
             writer.PerformPuts();
+            writer.EndStep();
```

Comment:
Same comment as for `examplePluginEngineRead.cpp`

Reply:
Fixed